usaspending_api/awards/models.py

```diff
@@ -271,3 +275,46 @@ class FinancialAssistanceAward(AwardAction):
     class Meta:
         managed = True
         db_table = 'financial_assistance_award'
+
+class SubAward(models.Model):
```

Comment:
@kaitlin Did we want to have a separate sub-award model, or just use the regular award model and have it link to other awards as a parent?